models/tridimensional/data_kleinstiver/Analysis of PyRosetta Output.py

```diff
@@ -0,0 +1,72 @@
+
+import scipy, scipy.stats.kendalltau, scipy.stats.chisquare
+
+def los_clnr(los):
+    """
+    Consumes a list of strings containing newline characters and commas, and 
+    cleans the strings by removing the newline characters and splitting by 
+    commas before returning a list of lists, where each list corresponds to
+    the important data from each string
+    """
+    newline_remove = map(lambda (el): el.strip(), los)
+    comma_split = map(lambda (el): el.split(','), newline_remove)
+    as_ints = [comma_split[0]] + map(lambda (el): map(float, el), comma_split[1:])
+    return as_ints
+
+def sort_data(lst):
```

Comment:
`data_sort(data_parsed)` is more clear (if you choose to go with data_parser above)